scripts/Jenkinsfile

```diff
@@ -41,43 +46,50 @@ node {
             deleteDir()
             checkout([
               $class: 'GitSCM',
-              branches: [[name: "*/${branch}"]],
-              extensions: scm.extensions + [[$class: 'CleanCheckout']],
+              branches: [
+                [name: "*/${branch}"]
+              ],
+              extensions: scm.extensions + [
+                [$class: 'CleanCheckout']
+              ],
               userRemoteConfigs: repo
             ])
           }
-          dir ("./") {
+          dir("./") {
             try {
               stage('Configure and Build') {
                 if (env.AWS_SSH_PEM_KEY && env.AWS_SSH_KEY_NAME) {
-                  dir("./tests/acceptance/modules/config/.ssh") {
+                  dir("./modules/config/.ssh") {
```

Comment:
shy , is this path correct?

Reply:
my $0.02, can we have the `.ssh` folder in the root path? Easier to isolate the keys and ignore `/distros-test-framework/.ssh/`?

Reply:
updated